src/test/java/org/openstreetmap/atlas/tags/annotations/extraction/AltitudeExtractorTest.java

```diff
@@ -0,0 +1,43 @@
+package org.openstreetmap.atlas.tags.annotations.extraction;
+
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Altitude;
+
+/**
+ * Unit test for {@link AltitudeExtractor}.
+ *
+ * @author bbreithaupt
+ */
+public class AltitudeExtractorTest
+{
+    @Test
+    public void validNumberTest()
+    {
+        Assert.assertEquals(Optional.of(Altitude.meters(20.5)),
+                AltitudeExtractor.validateAndExtract("20.5"));
+    }
+
+    @Test
+    public void validMetersTest()
```

Comment:
Does the extractor handle `20m` ?

Reply:
No it does not, as that is not valid in OSM.